app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,49 @@
   </head>
 
   <body>
+  <nav class="navbar navbar-expand-lg navbar-light style="background-color: #e3f2fd;">
+
+    <h1><a class="navbar-brand" href="#">Obituary Hub</a></h1>
+    <nav class="navbar-right navbar-light bg-light">
+      <form class="form-inline">
+        <% if current_user %>
+          <button class="btn btn-sm btn-outline-secondary" type="button">Logout</button>
+        <% else %>
+          <button class="btn btn-sm btn-outline-secondary" type="button">Login</button>
+          <button class="btn btn-sm btn-outline-secondary" type="button">Register</button>
```

Comment:
Looks good! We will figure out how to get these to the right of the page.

Reply:
![](https://media3.giphy.com/media/l0ErOholJjSmFlMFG/giphy.gif?cid=5a38a5a2s6hh3yo4clsp5rro27awslf1ka5n731ommoetwvi&rid=giphy.gif)